examples/specs/window_transform_movie_mean_difference.vl.json

```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+    "description": "Bar graph showing how each movie is different from the average movie rating",
+    "width": 300,
+    "height": 50,
+    "data": { "url": "data/movies.json" },
+    "layer": [{
+        "transform": [{
+                "window": [{
+                    "op": "mean",
+                    "field": "IMDB_Rating",
+                    "as": "AverageRating"
+                }],
+                "frame": [null, null]
+            },
+            {
+                "calculate": "datum.IMDB_Rating - datum.AverageRating",
+                "as": "DifferenceFromMean"
+            }
+        ],
+        "mark": { "type": "bar", "clip": true },
+        "encoding": {
+            "x": {
+                "field": "Title",
```

Comment:
I think there are definitely too many titles here to be comprehensible  -- pick a subset of 10 names using `filter`?

Reply:
Btw, this one is a clearly reasonable example for "### Different from Mean"  Example in the docs.

(You can remove different from total -- diff from mean is way more common)

Reply:
Idea:  try to pick titles that are academic award winners?

Reply:
Tried implementing this, by only displaying movies where the rating was 2.5 points higher than the average rating. This significantly reduced the number of titles being displayed.